Documentation/templates/MigrationGuide.md

```diff
@@ -1,23 +1,121 @@
-# Master
+# SwiftGen 6.0 Migration Guide
 
-*** Work In Progress — Will be consolidated once the release is ready ***
+If you're moving from SwiftGen 5.x to SwiftGen 6.0, you'll need to be aware that some templates have been renamed, removed or merged with others. Follow this Migration Guide to use the new name for SwiftGen 6.x of the templates you were previously using with SwiftGen 5.x.
+
+## Deprecated templates in SwiftGen 6.0
+
+### Colors
+
+| Old | New | Reason |
+| --- | --- | ------ |
+| `literals-swift3` | ✅ `literals-swift3` | |
+| `literals-swift4` | ✅ `literals-swift4` | |
+| `swift2` | ❌ _deleted_ | Really old Swift version |
+| `swift3` | ✅ `swift3` | |
+| `swift4` | ✅ `swift4` | |
+
+### Fonts
+
+| Old | New | Reason |
+| --- | --- | ------ |
+| `swift2` | ❌ _deleted_ | Really old Swift version |
+| `swift3` | ✅ `swift3` | |
+| `swift4` | ✅ `swift4` | |
+
+### Storyboards / IB
+
+The templates have been split up into separate templates for each specific functionality, in preparation of future functionalities such as accessibility identifiers.
+- One template to generate scene information.
+- One template to generate segue information.
+
+| Old | New | Reason |
+| --- | --- | ------ |
+| `swift2` | ❌ _deleted_ | Really old Swift version |
+| `swift3` | ➡️ `scenes-swift3`/`segues-swift3` | Split up into separate templates for the scenes and segues functionality |
+| `swift4` | ➡️ `scenes-swift4`/`segues-swift4` | Split up into separate templates for the scenes and segues functionality |
+
+⚠️ The `storybards` subcommand from SwiftGen 5.x has been renamed `ib` in SwiftGen 6.0, so be sure to put those templates in a `ib` subfolder and not an `storyboards` subfolder.
```

Comment:
Also be sure to read the paragraph in [other Migration Guide] about how to generate multiple outputs (from multiple templates) for a single input (a single set of input IB files).